### PR TITLE
add table scrollable

### PR DESCRIPTION
**Pull request by wangcansunking**
@wiredearp @zdlm @sampi @Chris-Xie 

Fixes issue # https://github.com/Tradeshift/tradeshift-ui/issues/569

Still have bugs, I try to fix before it is can be merged.

### CODE REVIEW

File: docs/dist/sri.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   },
   "@css": {
     "hashes": {
-      "sha384": "HqCjaTj5L/g9rFPonrcDX6VXdGUz/NWKqPh8jJVmE927AZkL2vsqsnqGlzuSy8PZ"
+      "sha384": "fIDL6WmWBR+X5nC00NuAFNwWirWRRzUjH0J1OSgG9gYEN0EsszdYMtrkMiEEBHiH"
     },
-    "integrity": "sha384-HqCjaTj5L/g9rFPonrcDX6VXdGUz/NWKqPh8jJVmE927AZkL2vsqsnqGlzuSy8PZ",
-    "path": "public/ts-10.0.11.min.css"
+    "integrity": "sha384-fIDL6WmWBR+X5nC00NuAFNwWirWRRzUjH0J1OSgG9gYEN0EsszdYMtrkMiEEBHiH",
+    "path": "dist/ts.css"
```
Comment (Chris-Xie) [inline docs/dist/sri.json]: Are these changes required？

**Comment by Chris-Xie** [comment]
Maybe we should make table content scrollable as the default behaviour rather than having to call `scrollable` explicitly?

File: docs/dist/sri.json (hunk 1 above)
Comment (wangcansunking) [reply docs/dist/sri.json]: I need to follow the develop workflow of the `tradeshift-ui`. But I don't know what it is....

**Comment by wiredearp** [comment]
@Chris-Xie If you have created your own Pager, I believe that the Table will *not* automatically attempt to page the data :information_source:

**Comment by wiredearp** [comment]
```
Maybe we should make table content scrollable as the default behavior when table component has a fixed height rather than having to call scrollable explicitly?
```

* The Table doesn't have a fixed height (in the offical APIs). If you feed it 2.000 rows, it may become 20.000 pixels high, but that height is not "fixed" and so the Table should not display a scrollbar.

* It may however have a fixed amount of rows if you call the `max` method. But again, it is not the pixels height that you specify here, it is the amount of rows at which the Table will break the data into multiple pages via the Pager. So again, the Table should not show a scrollbar because the height is not fixed.

* You can also use `size` method as an alternative to `max`. The difference is then that `size` will stretch the Table to the given amount of (standard) rows even before you feed data to the Table. If you do this, and if you specify `wrap: true` on the columns, you might see a scrollbar in the Table if and when some cells are wrapped to multiple lines. But in this case the header is indeed fixed while the body is scrolling.

However: When you use the `maximize` method, the trick is to *not* use any of these settings at all. That's why we really need to sit down and write the documentation for this layout mode.

```
Or we should tweak the maximize method so that it won't automatically paging the rows so that we can achieve the same goal with that?
```

I believe that your specific Table is only auto-paging the rows because you have the `max` number of rows specified. Try to not call the method. Did it help :question:

File: docs/dist/sri.json (hunk 1 above)
Comment (wiredearp) [reply docs/dist/sri.json]: It's related to the #subresource-integrity (we have a Slack channel for that project), but it is not actually working; we are computing the wrong values, so our build code will need to be fixed, but that is at least the general idea. This file will indeed be updated in every PR from now on, because it computes a hash for all the code in the bundled file; and the PR just changed the code.

**Comment by wiredearp** [comment]
:+1:
